scripts/crypt3.sh

```diff
@@ -2,115 +2,112 @@
 # Copyright 2006 Steve Dekorte, all rights reserved
 
 usage () {
-	echo "usages: "
-	echo "	crypt3.sh [path to openssl] -decrypt -delete key path"
-	echo "	crypt3.sh [path to openssl] -decrypt -keep   key path"
-	echo "	crypt3.sh [path to openssl] -encrypt -delete key path"
-	echo "	crypt3.sh [path to openssl] -encrypt -keep   key path"
-	exit 1
+    echo "Usage: "
+    echo "    $0 -decrypt -delete key path"
+    echo "    $0 -decrypt -keep   key path"
+    echo "    $0 -encrypt -delete key path"
+    echo "    $0 -encrypt -keep   key path"
+    exit 1
 }
 
-opensslexe=${1}
-command=${2}
-shred=${3}
-key=${4}
-filePath=${5}
-skipChecks=${6}
+if [[ $# != 4 ]]; then
+    usage
+fi
+
+opensslexe=`which openssl`
+command=${1}
+shred=${2}
+key=${3}
+filePath=${4}
+skipChecks=${5}
```

Comment:
Sorry, I meant the skipChecks parameter.. before (the parameter number 6) was not in the usage documentation.. now is the number 5 but still not existent.. right?